requirements-dev.txt

```diff
@@ -1,3 +1,4 @@
+matplotlib # Required for visualization in the Fourier module
```

Comment:
Is this relevant to decompositions?

Reply:
No! It's just a missing dep I found while testing.. removed this from the PR